pkg/graveler/model.go

```diff
@@ -35,6 +36,52 @@ func TagPartition(repoID RepositoryID) string {
 	return repoID.String()
 }
 
+func CommitPath(commitID CommitID) string {
+	return kv.FormatPath(commitsPrefix, commitID.String())
+}
+
+func CommitPartition(repoID RepositoryID) string {
```

Comment:
CommitPartition is misleading as it may lead the user to believe it's a different partition than branches for example.

Reply:
CommitPartition and TagPartition are both redundant - you should use the up-to-date RepoPath from master